src/pages/NotePageView.tsx

```
@@ -86,12 +115,101 @@ function NotePageEditorEditable({
 
+  // 最新の persistTitle をミュータブルな ref に退避する。useMutation の戻り値は
+  // 状態遷移（idle → pending → success）ごとに identity が変わるため、直接
+  // useCallback の依存に入れるとアンマウント用 effect の cleanup が過剰発火し、
+  // 保留中の保存が debounce を待たずに走ってしまう。ref 経由なら effect 側は
+  // 安定した参照だけを見て済む。
+  // Keep the latest persistTitle in a mutable ref. `useMutation()` returns a new
+  // object on every state transition (idle → pending → success), so referencing
+  // the mutation directly in a `useCallback` dep array would cause the unmount
+  // flush effect's cleanup to fire mid-typing and flush the debounce early.
+  const persistTitleRef = useRef<(nextTitle: string) => Promise<void>>(async () => {});
+  persistTitleRef.current = async (nextTitle: string) => {
```

ESLint

[error] Error: Cannot access refs during render

React refs are values that are not needed for rendering. Refs should only be accessed outside of render, such as in event handlers or effects. Accessing a ref value (the `current` property) during render can cause your component not to update as expected (https://react.dev/reference/react/useRef).

/home/jailuser/git/src/pages/NotePageView.tsx:128:3
  126 |   // flush effect's cleanup to fire mid-typing and flush the debounce early.
  127 |   const persistTitleRef = useRef<(nextTitle: string) => Promise<void>>(async () => {});
> 128 |   persistTitleRef.current = async (nextTitle: string) => {
      |   ^^^^^^^^^^^^^^^^^^^^^^^ Cannot update ref during render
  129 |     try {
  130 |       await updatePageMutation.mutateAsync({
  131 |         pageId: page.id,

(react-hooks/refs)